spec/support/page_helpers.rb

```diff
@@ -1,8 +1,8 @@
 # frozen_string_literal: true
 
 module PageHelpers
-  def reload_page_until_timeout!(text: '')
-    Timeout.timeout(Settings.timeouts.workflow) do
+  def reload_page_until_timeout!(text: '', num_seconds: Settings.timeouts.workflow)
+    Timeout.timeout(num_seconds) do
```

Comment:
the text to speech process takes a good while, this let's us up the timeout setting for just the one spot that takes a long time, leaving the default configured time everyone else